companies/3dhubs.json

```diff
@@ -7,7 +7,7 @@
         "commerce"
     ],
     "name": "3D Hubs B.V.",
-    "address": "Danzigerkade 23A\n1013 AP Amsterdam\nThe Netherlands",
+    "address": "3D Hubs B.V.\nDanzigerkade 23A\n3D Hubs B.V.\n1013 AP Amsterdam\nThe Netherlands",
```

Comment:
🚫 **[recordlint]** <[name-in-address](https://github.com/datenanfragen/data#addresses)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record includes `name` in the `address`.
```suggestion
    "address": "Danzigerkade 23A\n1013 AP Amsterdam\nThe Netherlands",
```